tenders/hvt/hvt_module_blk.c

```diff
@@ -69,7 +71,14 @@ static void hypercall_block_write(struct hvt *hvt, hvt_gpa_t gpa)
 
     ret = pwrite(e->b.hostfd, HVT_CHECKED_GPA_P(hvt, wr->data, wr->len),
             wr->len, pos);
-    assert(ret == wr->len);
+    if (ret != wr->len) {
+        if (ret == -1)
+            fprintf(stderr, "Fatal: error %d when writing\n", errno);
```

Comment:
Would a "%s" and "strerror(errno)" be suitable? So we don't need to decode the errno value manually.